packages/ui/components/combobox/src/LionCombobox.js

```diff
@@ -201,8 +202,24 @@ export class LionCombobox extends LocalizeMixin(OverlayMixin(CustomChoiceGroupMi
 
   reset() {
     super.reset();
-    // @ts-ignore _initialModelValue comes from ListboxMixin
-    this.value = this._initialModelValue;
+    if (!this.multipleChoice) {
+      // @ts-ignore _initialModelValue comes from ListboxMixin
+      this.value = this._initialModelValue;
```

Comment:
If we set the modelValue, it triggers the "main loop". Then the modelValue is synced with the "presentation" (how the individual options look and their a11y attrs are handled)

Reply:
It now added the value to the inputNode.Value, which is not needed for multipleChoice.